packages/docusaurus-plugin-content-docs/src/create.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import logger from '@docusaurus/logger';
+import {
+  getFileCommitDate,
+  FileNotTrackedError,
+  GitNotFoundError,
+} from '@docusaurus/utils';
+
+let showedGitRequirementError = false;
+let showedFileNotTrackedError = false;
+
+export async function getFileCreate(
```

Comment:
Wouldn't it make sense to group getFileLastUpdate + getFileCreate into the same file?

cc @Josh-Cena I thought we had something like gitUtils here in the past